lib/open_project/backlogs/engine.rb

```diff
@@ -129,7 +129,8 @@ def self.settings
     )
 
     patches [:PermittedParams, :WorkPackage, :Status, :MyController, :Project,
-      :ProjectsController, :ProjectsHelper, :Query, :User, :VersionsController, :Version]
+      :ProjectsController, :ProjectsHelper, :Query, :User, :VersionsController, :Version,
```

Comment:
Line is too long. [89/80]<br>Align the elements of an array literal if they span more than one line.